### PR TITLE
Add ability to run in Privileged mode to Na_ontap_command

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
New Option to allow running in privileged mode 

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
- na_ontap_command.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/storage/netapp/na_ontap_command.py [hunk 1]
```diff
@@ -88,18 +114,19 @@ def run_command(self):
                                   exception=traceback.format_exc())
 
     def apply(self):
+        ''' calls the command and returs raw output '''
```
Comment (lonico) [inline lib/ansible/modules/storage/netapp/na_ontap_command.py]: typo: returns

File: test/units/modules/storage/netapp/test_na_ontap_command.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+from units.compat.mock import patch, Mock
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+import ansible.module_utils.netapp as netapp_utils
+
+from ansible.modules.storage.netapp.na_ontap_command \
+    import NetAppONTAPCommand as my_module  # module under test
+
+if not netapp_utils.has_netapp_lib():
+    pytestmark = pytest.skip('skipping as missing required netapp_lib')
```
Comment (lonico) [inline test/units/modules/storage/netapp/test_na_ontap_command.py]: pytest.mark.skip

**Review by lonico — CHANGES_REQUESTED** [review]

**Review by lonico — APPROVED** [review]
shipit

File: lib/ansible/modules/storage/netapp/na_ontap_command.py [hunk 3]
```diff
@@ -65,19 +77,33 @@ def __init__(self):
...
         if HAS_NETAPP_LIB is False:
             self.module.fail_json(msg="the python NetApp-Lib module is required")
         else:
             self.server = netapp_utils.setup_na_ontap_zapi(module=self.module)
 
-    def run_command(self):
+    def asup_log_for_cserver(self, event_name):
+        """
+        Fetch admin vserver for the given cluster
+        Create and Autosupport log event with the given module name
```
Comment (thedoubl3j) [inline lib/ansible/modules/storage/netapp/na_ontap_command.py:90]: nit pick `and` -> `an`

**Review by thedoubl3j — APPROVED** [review]
looks good to me, only a small nit pick that isn't a blocker.

**Comment by gundalow** [comment]
Merged, thanks.
Typo was just in comment, so I'm OK with this as is.

**Pull request merged**